style.css

```diff
@@ -518,3 +538,222 @@ textarea {
   place-items: center;
   margin-top: 20px;
 }
+
+@media screen and (min-width: 768px) {
+  .cont-list {
+    display: flex;
+    gap: 55px;
+    margin-right: 45px;
+    list-style: none;
+    margin-left: 60%;
+  }
+
+  .cont-list li a {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .nav-logo {
+    margin-left: 10%;
+  }
+
+  /* .introduction p{
+  width: 50%;
+} */
+
+  .hambuger {
+    display: none;
+  }
+
+  .pic {
+    background-image: url("images/bg.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding-top: 249px;
+    padding-left: 260px;
+    padding-right: 260px;
+  }
+
+  .intro-card {
+    padding-left: 1%;
+  }
+
+  .work-card {
+    flex-direction: row;
+    width: 750px;
+    gap: 30px;
+    justify-content: space-between;
+  }
+
+  .work-card img {
+    width: 300px;
+    height: 300px;
+    object-fit: contain;
+  }
+
+  .work-card:nth-child(even) {
+    flex-direction: row-reverse;
+  }
+
+  .grid {
+    justify-content: center;
+    flex-direction: row;
+    gap: 30px;
+  }
+
+  .icon2 {
+    flex-direction: column;
+  }
+
+  /* .languages_details {
+    display: flex;
+    text-align: center;
+    gap: 20px;
+    flex-wrap: wrap;
+    border-radius: 8px;
+    padding: 0.5em;
+    background-color: #f7f7f9;
+    margin-top: 0.5em;
+  } */
+
+  .lang-item0 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    color: #01f;
+    width: 130px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    flex: none;
+    order: -0;
+    flex-grow: 0;
+  }
+
+  .contact1 {
+    background-image: url(images/Desktop\ Contact.png);
+    background-repeat: no-repeat;
+    background-size: contain;
+    padding-top: 249px;
+    align-items: center;
+  }
+
+  .contact {
+    align-items: center;
+    align-self: center;
+    align-content: center;
+  }
+
+  .first_me {
+    align-items: center;
+  }
+
+  .first {
+    display: block;
+    order: 1;
+  }
+
```

Comment:
- Please, let's kindly improve the width of the form paragraph to match the Figma design as much as we can as your paragraph now is not having the correct width size. See the images below to improve your design:


*Figma*
![Screenshot from 2023-02-23 22-25-19](https://user-images.githubusercontent.com/59397159/221034219-4ac408ea-2515-4016-872e-1ed8f3455738.png)

*Your design*
![Screenshot from 2023-02-23 22-24-46](https://user-images.githubusercontent.com/59397159/221034224-f8f52473-79d1-4299-bb14-8606e5d027d5.png)